tests/schema/models.py

```diff
@@ -139,6 +138,14 @@ class Meta:
         db_table = "schema_book"
 
 
+class BookWithTags(models.Model):
+    title = models.CharField(max_length=100, db_index=True)
+    tags = models.ManyToManyField('TagM2MTest', related_name='books')
+
+    class Meta:
+        apps = new_apps
```

Comment:
This model is used only in a single test so we can add an inline model directly in the `test_alter_field_attributes_doesnt_perform_queries_m2m`.